src/components/controls/MediaStreamControl.tsx

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface IMediaStreamControlProps {
+    getAudioEnabled: () => boolean;
+    setAudioEnabled: (enabled: boolean) => void;
+    getVideoEnabled: () => boolean;
+    setVideoEnabled: (enabled: boolean) => void;
+}
```

Comment:
The interface should be renamed. See the comment above. Also, the interface should use values over getter functions. This will trigger a re-render for the renderer.

```tsx
interface IMediaStreamControlRendererProps {
     audioEnabled: boolean;
     setAudioEnabled: (enabled: boolean) => void;
     videoEnabled: boolean;
     setVideoEnabled: (enabled: boolean) => void;
}
```

Reply:
Done.